website/docs/brain/custom-applications.md

```diff
@@ -73,20 +77,71 @@ ssh amiga
 exit
 ```
 
-### Select an app
+## Prepare an app
+
+Follow the instructions below to select an example from the [`farm-ng-amiga` examples](https://github.com/farm-ng/farm-ng-amiga/tree/main/py/examples) or create an app from the [`amiga-app-template`](https://github.com/farm-ng/amiga-app-template) repository.
 
-Select an example from the [amiga-brain-api examples](https://github.com/farm-ng/amiga-brain-api/tree/main/py/examples) or create an app from the [template repository](https://github.com/farm-ng/amiga-brain-example).
-See the [template README](https://github.com/farm-ng/amiga-brain-example#readme) for instructions on cloning and creating an app.
+### Select an app
 
 :::caution coming soon
-The [amiga-brain-api examples](https://github.com/farm-ng/amiga-brain-api/tree/main/py/examples) will soon be updated to match
+The folder structure of the examples in [farm-ng-amiga examples](https://github.com/farm-ng/farm-ng-amiga/tree/main/py/examples) will soon be updated to match
 the template repository for setup / install on the amiga.
 
-In the meantime, you can copy the `main.py`, `requirements.txt`, and `assets/` directory (if present)
-from the example and drop them into an app you create in the template repository.
+In the meantime, follow the instructions to [Create an app](#create-an-app).
+You can then copy the `main.py`, `requirements.txt`, and `assets/` directory (if present)
+from the example and drop them into an app you create with the template repository.
 :::
 
-### Live Code Syncing
+### Create an app
+
+Use the [`amiga-app-template`](https://github.com/farm-ng/amiga-app-template) repository to create your own custom application with the following instructions.
+
+#### Create a repository from the template
+
+Navigate to the [`amiga-app-template`](https://github.com/farm-ng/amiga-app-template) repository on GitHub.
+
+Click on green *Use this template* button (top right) to create a new repository based on this repo
```

Comment:
add something like this here

![template_app](https://user-images.githubusercontent.com/5157099/204756343-6b572dc7-076c-4c2e-b09b-fe593672f8fa.gif)

Reply:
Feel free